source/NVDAHelper.py

```diff
@@ -78,50 +79,68 @@ def nvdaController_speakText(text):
 	return SystemErrorCodes.SUCCESS
 
 
+# C901 'nvdaController_speakSsml' is too complex
+# Note: when working on nvdaController_speakSsml, look for opportunities to simplify
+# and move logic out into smaller helper functions.
 @WINFUNCTYPE(c_long, c_wchar_p, c_int, c_int, c_bool)
-def nvdaController_speakSsml(
+def nvdaController_speakSsml(  # noqa: C901
 		ssml: str,
-		symbolLevel: int,
+		symbolLevel: "SymbolLevel",
 		priority: "SpeechPriority",
 		asynchronous: bool,
-):
+) -> int | SystemErrorCodes:

```

Comment:
I think since the `-1` value was removed this is the return type now?

```suggestion
) -> SystemErrorCodes:
```